services/src/main/java/org/apache/druid/cli/CliPeon.java

```diff
@@ -259,6 +259,8 @@ public void configure(Binder binder)
             // configuration based locations.  This will override them anyway.  This is also stopping
             // configuration of other parameters, but I don't think that's actually a problem.
             // Note, if that is actually not a problem, then that probably means we have the wrong abstraction.
+            // FIXME: Is this actually necessary after this PR, or was this just a workaround for
```

Comment:
I don't think it's necessary after this PR, but would be worth to double check.

Reply:
OK. Do you think just removing this statement and running integration tests would be enough to double check?

Reply:
Yeah, I think it would be enough.